build.sbt

```diff
@@ -57,9 +57,8 @@ val AkkaPersistenceCassandraVersion = "0.102"
 val PrometheusClientVersion = "0.6.0"
 val ScalaTestVersion = "3.0.5"
 val ProtobufVersion = "3.9.0"
-val GraalVersion = "19.3.0"
-val DockerBaseImageVersion = "openjdk:8-jre-slim-buster"
-val svmGroupId = if (GraalVersion startsWith "19.2") "com.oracle.substratevm" else "org.graalvm.nativeimage"
+val GraalVersion = "20.0.0"
+val DockerBaseImageVersion = "openjdk:8-jre-buster"
```

Comment:
@srikanthops We need to use the full, not the slim, jre to make sure that SSL etc works.